src/App/Utilities/AppHelpers.cs

```diff
@@ -382,6 +383,17 @@ public static async Task<bool> IsHideEmailDisabledByPolicyAsync()
                 policy => policy.Data.ContainsKey("disableHideEmail") && (bool)policy.Data["disableHideEmail"]);
         }
 
+        public static async Task<bool> IsShowingVaultFilterAsync()
+        {
+            var organizationService = ServiceContainer.Resolve<OrganizationService>("organizationService");
+            var policyService = ServiceContainer.Resolve<IPolicyService>("policyService");
+            
+            var organizations = await organizationService.GetAllAsync();
+            var personalOwnershipPolicyApplies =
+                    await policyService.PolicyAppliesToUser(PolicyType.PersonalOwnership);
+            return (organizations?.Any() ?? false) && !personalOwnershipPolicyApplies;
+        }
```

Comment:
I personally don't like much the static methods on AppHelpers that can be placed on a service or somewhere else.
In this case I'd just add this method to the `OrganizationService`. I think when we add unit testing it's better to have these in non-static places in order to be able to mock them easier and we don't strictly need to resolve each service in each call to the method (as a matter of fact placing it in the `OrganizationService` saves us one `Resolve` call from the service container) 😄

Reply:
I ended up using `PolicyService` to prevent a circular dependency when constructing `OrganizationService`